api/src/main/java/io/minio/messages/AndOperator.java

```diff
@@ -67,10 +73,50 @@ public AndOperator(
     this.tags = (tags != null) ? Collections.unmodifiableMap(tags) : null;
   }
 
+  public AndOperator(
```

Comment:
Add new arguments at the end like
```java
  public AndOperator(
      @Nullable @Element(name = "Prefix", required = false) String prefix,
      @Nullable
          @ElementMap(
              attribute = false,
              entry = "Tag",
              inline = true,
              key = "Key",
              value = "Value",
              required = false)
          Map<String, String> tags,
      @Element(name = "ObjectSizeLessThan", required = false) Integer objectSizeLessThan,
      @Element(name = "ObjectSizeGreaterThan", required = false) Integer objectSizeGreaterThan) {
    this(prefix, tags);
    this.objectSizeLessThan = objectSizeLessThan;
    this.objectSizeGreaterThan = objectSizeGreaterThan;
  }
```

This applicable to all constructors in the PR